### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by liushunyu** [DSE-MSU/DeepRobust#35]
I find that you use `nll_loss` as the loss function in PGDtraining, why you don't use `cross_entropy`? 

In ResNet, you use the `cross_entropy` to train the model

**Comment by liushunyu** [comment]
Whether the same effect can be achieved with these two loss functions？

**Comment by liushunyu** [comment]
So I think that you should use `cross_entropy ` as the loss function in PGDtraining?

Because there is not `log_softmax` in your `resnet`, `vgg`, `densenet` and so on.
